srml/aura/src/lib.rs

```diff
@@ -166,19 +151,53 @@ impl HandleReport for () {
 pub trait Trait: timestamp::Trait {
 	/// The logic for handling reports.
 	type HandleReport: HandleReport;
+
+	/// The identifier type for an authority.
+	type AuthorityId: Member + Parameter + Default;
 }
 
 decl_storage! {
 	trait Store for Module<T: Trait> as Aura {
 		/// The last timestamp.
 		LastTimestamp get(last) build(|_| 0.into()): T::Moment;
+
+		/// The current authorities
+		pub Authorities get(authorities) config(): Vec<T::AuthorityId>;
 	}
 }
 
 decl_module! {
 	pub struct Module<T: Trait> for enum Call where origin: T::Origin { }
 }
 
+impl<T: Trait> Module<T> {
+	fn change_authorities(new: Vec<T::AuthorityId>) {
+		<Authorities<T>>::put(&new);
+
+		let log: DigestItem<T::Hash> = DigestItem::Consensus(AURA_ENGINE_ID, new.encode());
```

Comment:
Wouldn't it be better to declare something like:
```
#[derive(Decode, Encode)]
enum ConsensusEvent {
   AuthoritiesChange(Vec<T::AuthorityId>),
}
```
and make log contains `DigestItem::Consensus(AURA_ENGINE_ID, ConsensusEvent::AuthoritiesChange(new).encode())` instead? I'm just worrying that it'll be hard to change if we would decide to emit something else using `DigestItem::Consensus` in the future.